database/factories/UserFactory.php

```diff
@@ -1,23 +1,33 @@
 <?php
 
-use Faker\Generator as Faker;
+namespace Database\Factories;
 
-/*
-|--------------------------------------------------------------------------
-| Model Factories
-|--------------------------------------------------------------------------
-|
-| This directory should contain each of the model factory definitions for
-| your application. Factories provide a convenient way to generate new
-| model instances for testing / seeding your application's database.
-|
-*/
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
 
-$factory->define(App\User::class, function (Faker $faker) {
-	return [
-		'name' => $faker->name,
-		'email' => $faker->unique()->safeEmail,
-		'password' => '$2y$10$TKh8H1.PfQx37YgCzwiKb.KjNyWgaHb9cbcoQgdIVFlYg7B77UdFm', // secret
-		'remember_token' => str_random(10),
-	];
-});
+class UserFactory extends Factory
+{
+	/**
+	 * The name of the factory's corresponding model.
+	 *
+	 * @var string
+	 */
+	protected $model = User::class;
+
+	/**
+	 * Define the model's default state.
+	 *
+	 * @return array
+	 */
+	public function definition()
+	{
+		return [
+			'name' => $this->faker->name,
+			'email' => $this->faker->unique()->safeEmail,
+			'email_verified_at' => now(),
+			'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
```

Comment:
What the hell is this hardcoded password? Yeah, I know it was there in the old version as well...

Reply:
it is a user Factory to use with seeding, we do not care as this is used for testing in a way that we are not doing.